### ISSUE TITLE
What is the real benefit of this system?

### ISSUE THREAD

**Issue by rohit-kugve** [osnetsvn/span#13]
SOCC 2016 Review #192A
Overall merit: 1. Strong reject
Reviewer expertise: 3. Knowledgeable

You badly need stronger examples.  All three of the things you describe have been built over existing hypervisors and it is not clear how your system does anything other than make them more complicated and less stable.  Moreover, the examples, as read-only monitoring tools, do not demonstrate that the system is safe or functional for tasks that require mutating guest state, changing scheduling, modifying file or network data, etc.

In summary, this paper doesn't quite seem cooked enough at this point.  It has some hair-raisingly exciting mechanism, but doesn't have the examples to make me want it, or even to make me believe that it works.  I encourage you to think hard about those examples, and to focus on what the right APIs and developer usability concerns need to be for this system to be viable.
